schema/project-level/project-schema.json

```diff
@@ -180,8 +180,26 @@
   "definitions": {
     "ContractingProcess": {
       "title": "Contracting process",
-      "description": "Within the project level data specification, a contracting process provides both summary information, and a log of changes over time, either manually curated, or automatically generated through linked OCDS releases. Where OCDS data is available, most ContractingProcess summary fields can be derived from OCDS releases, although the exact method to derive data may vary between implementations.",
+      "description": "Within the project level data specification, a contracting process provides both summary information, and a log of changes over time, either manually curated, or automatically generated through linked OCDS releases.",
       "type": "object",
+      "properties": {
+        "summary": {
+          "$ref": "#/definitions/Summary"
+        },
+        "releases": {
+          "title": "Release index",
+          "description": "The information that known about a contracting process changes over time, both as new information becomes available, and as changes are made (such as amendments to scope and value). In the OCDS, each new update of information is known as a 'release'. \n\n This section provides space to record a link to each available release, and to summarise any variations that occur in that release. Variations may be identified by comparing a new release to previous releases to check for relevant changes.\n\n In cases where OCDS data is not available, the `releaseList` can be used to manually record a log of changes, with the date and details of variations.",
```

Comment:
The first sentence is missing a word to be grammatical. 'the `releaseList`' should be 'this field'. American spelling is preferred (e.g. 'summarize' not 'summarise').